docs/configuration.asciidoc

```diff
@@ -617,3 +626,36 @@ The default unit for this option is `ms`
 Sets the logging level for the agent.
 
 Valid options: `Critical`, `Error`, `Warning`, `Info`, `Debug`, `Trace` and `None` (`None` disables the logging).
+
+[[config-all-options-summary]]
+=== All options summary
+
+[options="header"]
+|============
+| Option name | Is reloadable? | Keywords
+| <<config-capture-body,`CaptureBody`>> | Yes | HTTP, Performance
+| <<config-capture-body-content-types,`CaptureBodyContentTypes`>> | Yes | HTTP, Performance
+| <<config-capture-headers,`CaptureHeaders`>> | No | HTTP, Performance
+| <<config-central-config,`CentralConfig`>> | No | Core
+| <<config-environment,`Environment`>> | No | Core
+| <<config-flush-interval,`FlushInterval`>> | No | Reporter
+| <<config-global-labels,`GlobalLabels`>> | No | Core
+| <<config-log-level,`LogLevel`>> | No | Supportability
+| <<config-max-batch-event-count,`MaxBatchEventCount`>> | No | Reporter
+| <<config-max-queue-event-count,`MaxQueueEventCount`>> | No | Reporter
+| <<config-metrics-interval,`MetricsInterval`>> | No | Reporter
+| <<config-sanitize-field-names,`SanitizeFieldNames`>> | No | Core
+| <<config-secret-token,`SecretToken`>> | No | Reporter
+| <<config-server-urls,`ServerUrls`>> | No | Reporter
+| <<config-service-name,`ServiceName`>> | No | Core
+| <<config-service-version,`ServiceVersion`>> | No | Core
+| <<config-span-frames-min-duration,`SpanFramesMinDuration`>> | No | Stacktrace, Performance
+| <<config-stack-trace-limit,`StackTraceLimit`>> | No | Stacktrace, Performance
+| <<config-transaction-max-spans,`TransactionMaxSpans`>>  | Yes | Performance, Core
+| <<config-transaction-sample-rate,`TransactionSampleRate`>> | Yes | Performance, Core
+|============
```

Comment:
I really like this idea. My only concern is that it's one more spot in the documentation that needs to be updated if a config changes or a new one is added. Let's keep it around, just wanted to share that thought with you. It would be awesome if we could someday automate these jump tables.

Reply:
Yes, I agree if we get to a situation when maintaining this summary becomes too error-prone we will consider how to automate it.

Reply:
BTW if you think we should have a different set of columns please let me know.